### ISSUE TITLE
Accessibility - goal headings do not have sufficient contrast

### ISSUE THREAD

**Issue by shaneporter** [datasciencecampus/sdg-indicators#342]
Haven't looked at _all_ the goal headings but I suspect that many have insufficient contrast.

The current design uses the background and foreground colours of the goal logo, shown on the left, so _both_ fail accessibility guidelines. 

<img width="732" alt="screen shot 2017-09-26 at 10 09 13" src="https://user-images.githubusercontent.com/2493614/30852200-de0d2200-a2a2-11e7-9f28-b287c7468e73.png">

Pretty sure we aren't permitted to modify the UN logos, but we could tweak the text/background combo in the design.

**Comment by AnnCorp** [comment]
No we can't modify the logos (though US have used white on black versions in their reporting status page so presume that kind of thing ok and we could change to that style on that page if needed).
Have just been re-reading the user testing report and the issues flagged re page title etc - maybe need to reconsider layout more generally?
Have you got any suggestions on what text/background combos could provide sufficient contrast?

**Comment by shaneporter** [comment]
I'd use the online [Color Contrast Checker](https://webaim.org/resources/contrastchecker/), starting with the existing colour combinations. If they failed, I would adjust accordingly.

Minimum contrast may be easier to achieve with black text, but there are no doubt other designs that would work - I liked using the same colour for continuity, but I think a slight variation could be quite eye-pleasing. The white on black icons fix the accessibility issue, but it's a shame to lose the colour.

**Comment by AnnCorp** [comment]
Closing this ticket as its covered by 1060
